src/Form/index.tsx

```diff
@@ -341,6 +392,13 @@ function Form({
 
     setAutoValidate(false); // Each step to initially not auto validate
 
+    if (
+      formContextRef &&
+      Object.prototype.hasOwnProperty.call(formContextRef, 'current')
+    )
+      // Need to update the context because it has activeStep closure'd in
+      formContextRef.current = getFormContextHelper();
```

Comment:
I tried just setting .current once but I was getting a runtime error that `key` didn't exist on `newStep.key` when running 

```
setStep: (stepKey: any) => {
    changeStep(stepKey, newStep.key, props.steps, props.history);
  },
```

Reply:
another alt for naming would be to make this `ref.current.context = ....`